app/data/census_individual.json

```diff
@@ -2582,13 +2733,13 @@
                     "sections": [
                         {
                             "id": "job-description-section",
-                            "number": "34",
+                            "number": "37",
                             "title": "{{ [answers.first_name, answers.last_name] | format_household_name }}",
                             "questions": [
                                 {
                                     "id": "job-description-question",
                                     "title": "Briefly describe what you do (did) in your main job.",
-                                    "number": "34",
+                                    "number": "37",
```

Comment:
I think we agreed to keep this on the census household so I have sync'd with what that defines in the label is that correct @jonnyshaw89?